### ISSUE TITLE
Couldn't find implydata's request/request-promise-native forks

### ISSUE THREAD

**Issue by themaric** [implydata/plywood-druid-requester#36]
Hi,

I'm getting the following error when trying to bump `plywood-druid-requester` to the latest v3 release:

```
$ yarn install
yarn install v1.22.19
[1/4] Resolving packages...
error Couldn't find package "@implydata/request@^2.88.3" required by "plywood-druid-requester@^3.0" on the "npm" registry.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Error: Couldn't find package "@implydata/request-promise-native@^1.0.9" required by "plywood-druid-requester@^3.0" on the "npm" registry.
    at MessageError.ExtendableBuiltin (/usr/lib/node_modules/yarn/lib/cli.js:721:66)
    at new MessageError (/usr/lib/node_modules/yarn/lib/cli.js:750:123)
    at PackageRequest.<anonymous> (/usr/lib/node_modules/yarn/lib/cli.js:36590:17)
    at Generator.throw (<anonymous>)
    at step (/usr/lib/node_modules/yarn/lib/cli.js:310:30)
    at /usr/lib/node_modules/yarn/lib/cli.js:323:13
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

Not sure about the cause of the issue. Can you please shed some light on this?

Thanks!

**Comment by jgoz** [comment]
Ah, we probably need to make those packages public. Thanks for raising this.

**Comment by jgoz** [comment]
This should be resolved in 3.0.1, could you please try again?

**Comment by themaric** [comment]
It's resolved, thanks again!
